benchmark/testsystems/testsystems.py

```diff
@@ -37,14 +37,38 @@ def load_dhfr_explicit(constrained=True):
 
     return topology, system, positions
 
+def load_src_explicit(constrained=True):
+    if constrained:
+        constraints = app.HBonds
+        rigid_water = True
+    else:
+        constraints = None
+        rigid_water = False
+
+    testsystem = SrcExplicit(constraints=constraints, rigid_water=rigid_water)
+    topology, system, positions = testsystem.topology, testsystem.system, testsystem.positions
+
+    keep_only_some_forces(system)
+    add_barostat(system)
+
+    return topology, system, positions
+
 def load_dhfr_reaction_field(constrained=True):
     """DHFR in explicit solvent, but using reaction field instead of PME for nonbonded"""
 
-    topology, system, positions = load_dhfr_explicit(constrained)
+    if constrained:
+        constraints = app.HBonds
+        rigid_water = True
+    else:
+        constraints = None
+        rigid_water = False
+
 
-    system.getForces()
+    testsystem = DHFRExplicit(nonbondedMethod=app.CutoffPeriodic, constraints=constraints, rigid_water=rigid_water)
```

Comment:
You'll also want to use [this function](http://openmmtools.readthedocs.io/en/latest/api/generated/openmmtools.alchemy.AbsoluteAlchemicalFactory.html#openmmtools.alchemy.AbsoluteAlchemicalFactory.replace_reaction_field) to recode the reaction field electrostatics as a reaction-field with a switch (for energy conservation).

Note that you probably want to use a larger cutoff as well. Reaction field isn't really intended for short 9A cutoffs; you'll want to use something more like 12A or more.